canonical-web-and-design/jaas-dashboard
# Restyle primary navigation

## Done

- Remove top bar
- Move links into sidebar under toggle 
- Apply dark theme

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check code
- View nav

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/942
